azurelinuxagent/ga/update.py

```diff
@@ -941,31 +937,17 @@ def _download(self):
         uris_shuffled = self.pkg.uris
         random.shuffle(uris_shuffled)
         for uri in uris_shuffled:
-            if not HostPluginProtocol.is_default_channel() and self._fetch(uri.uri): # pylint: disable=R1723
+            if self._fetch(uri.uri): # pylint: disable=R1723
```

Comment:
NIT: Pylint

Reply:
Do we want to keep similar behaviour for this too? Basically going hostga first and then direct?